src/Regression/Adapter/SugarCRMAware.php

```diff
@@ -429,7 +429,11 @@ protected function detectSugarSubscription(): void
         $subscriptionRequest = new Request('GET', 'https://authenticate.sugarcrm.com/rest/subscription/' . $this->config->getLicense());
 
         $this
-            ->send($subscriptionRequest)
+            ->send($subscriptionRequest, [
+                'chromeClientOptions' => [
```

Comment:
We can pass here DTO instead of an array, so it will give us autocomplete in IDE and earlier validation, also it will eliminate `array => DTO` conversion step.